### PR TITLE
Panel/BarGauge: Prevent overflow in panel with many series

### LINKED ISSUE #24889: Edit Panel - Bar Gauge leaks the chart on the query editor if you have more than 32 times series
**What happened**:
I have more than 32 series in a chart. 
I created a Bar Gauge chart with Horizontal Orientation. The Panel visualization leaked on the query editor.

image

**What you expected to happen**:
Chart in Query editor should respect the real estate available for bar gauge in horizontal mode.

**How to reproduce it (as minimally and precisely as possible)**:
https://play.grafana.org/d/Fbp5uPsZk/1-new-features-in-v6-6?editPanel=5&orgId=1. Set the Series Count to 32 in Panel Editor.

**Anything else we need to know?**:
When you come out of the panel editor, dashboard shows the chart correctly. .

**Environment**:
- Grafana version: 7.0.0
- Data source type & version: Does not matter
- OS Grafana is installed on: Docker
- User OS & Browser: Chrome
- Grafana plugins: Bar Gauge
- Others:

### PR DESCRIPTION
**Pull request by kaydelaney**
**What this PR does / why we need it**:
Prevents bar gauges from overflowing panel when there are many series.

Before: overflow

After: scrollable

**Which issue(s) this PR fixes**:
Closes #24889

### CODE REVIEW

File: public/app/plugins/panel/bargauge/BarGaugePanel.tsx [hunk 1]
```diff
@@ -100,8 +100,10 @@ export class BarGaugePanel extends PureComponent<PanelProps<BarGaugeOptions>> {
         renderCounter={renderCounter}
         width={width}
         height={height}
+        minVizHeight={10}
         itemSpacing={this.getItemSpacing()}
         orientation={options.orientation}
+        repeaterStyle={{ overflow: 'hidden scroll' }}
```
Comment (dprokop) [inline public/app/plugins/panel/bargauge/BarGaugePanel.tsx:106]: Uhm, maybe this style could be set internally in the VizRepeater when `minVizHeight` is set?

**Review by dprokop — APPROVED** [review]
This looks good, left one nit to consider :)

File: public/app/plugins/panel/bargauge/BarGaugePanel.tsx (hunk 1 above)
Comment (kaydelaney) [reply public/app/plugins/panel/bargauge/BarGaugePanel.tsx:106]: Oh good idea, will do

**Comment by igorzavgorodniy** [comment]
hi @kaydelaney. is this fix get to release or still pending?

**Pull request merged**